web/server/parser/packageParser.go

```diff
@@ -101,10 +101,12 @@ func (self *outputParser) registerTestFunction() {
 }
 func (self *outputParser) recordTestMetadata() {
 	testName := strings.Split(self.line, " ")[2]
-	self.test = self.testMap[testName]
-	self.test.Passed = !strings.HasPrefix(self.line, "--- FAIL: ")
-	self.test.Skipped = strings.HasPrefix(self.line, "--- SKIP: ")
-	self.test.Elapsed = parseTestFunctionDuration(self.line)
+	if test, ok := self.testMap[testName]; ok {
```

Comment:
Sure, I can put something together. Oddly enough I think this bug was introduced as a result of this change: https://github.com/golang/go/issues/10594